src/transactions/MergeOpFrame.cpp

```diff
@@ -92,6 +92,23 @@ MergeOpFrame::doApply(Application& app, LedgerDelta& delta,
         return false;
     }
 
+    if (ledgerManager.getCurrentLedgerVersion() >= 10)
+    {
+        SequenceNumber maxSeqNum =
+            ledgerManager.getCurrentLedgerHeader().ledgerSeq;
+        maxSeqNum = (maxSeqNum << 32) - 1;
```

Comment:
I think a comment here explaining why this condition exists would be nice

Reply:
I forgot that I wanted to get rid of this magic across the code base